ConsoleApp6/IDecorater.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace KATA
+{
+    internal interface IDecorater:IPaymentsServices
+    {
+
+    }
```

Comment:
why the decorator is an interface? you add an extra un-needed step that way I think

Reply:
the decorator is an abstract class, containing a ref. to the behavioral interface it implements, with the needed functionalities